packages/web/test/mocks/algo-signer-mock.ts

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable sonarjs/no-identical-functions */
+import { unknown } from "zod";
+
+import { WallectConnectSession } from "../../src";
+import {
+	AlgoSigner,
+	Encoding,
+	JsonPayload,
+	MultisigTransaction,
+	RequestErrors,
+	Transaction,
+	WalletTransaction,
+} from "../../src/algo-signer-types";
+
+const suggestedParamsMock = {
```

Comment:
Shouldn't we also define the type of this? I think it's suggested params.

Reply:
Seem likes we can't use suggested params.